Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,304 @@
+    PointedGraph[Rdf](
+      to,
+      ops.makeGraph(pg.graph.triples.map{ triple =>
+        ops.fromTriple(triple) match {
+          case (oldNode,rel,obj) => ops.makeTriple(to,rel,obj)
+          case (subj,rel,oldNode) => ops.makeTriple(subj,rel,to)
+          case _ => triple
+        }
+      })
+    )
+  }
+}
+
+defined class PGwrapper
+```
+
+After this, one can simply change the header by into something simple by calling this function like so:
+
+```scala
+@ val finalKeyPG = keyPG.rename(URI("#key"))
+
+finalKeyPG: PointedGraph[Rdf] = org.w3.banana.PointedGraph$$anon$1@4353a749
+
+@ finalKeyPGPointer = finalKeyPG.pointer
+
+finalKeyPGPointer : Rdf#Node = #key
+
+@ finalKeyPGGraph = finalKeyPG.graph
+
+finalKeyPGGraph: Rdf#Graph =  {#key @http://www.w3.org/ns/auth/cert#modulus "0087acb657366c6a911a0ce2470c24b8d85dd21ae76c6001db37d122c2eafe0fe2ae35541ccdd3c1c81603d98dd7d61b2c31c8605f81fbc3566604b7755793698836caaa99d868477c46b5b735529b50c0acfea6e10fc9b953697a67d57499801beb651e7e08343131a00d1873ab753ce5e79fc961874ee132472f7f210bc38966081fe263c620b67469b52cc555a26dee4fa10d8f40959e0e13516cc0bd1c1669ce53367d28248149142429b127f01e13d9bf21de52a4ac5694d5038a94178e144823b152fa19cbbc094dd40ddbd41e2195a19081125887fccedae21c50660c629a6f2ba4c85a54e1a6472f90b7b62ac7fe58fdae9daa0edc8edec49802fbfac7"^^http://www.w3.org/2001/XMLSchema#hexBinary; #key @http://www.w3.org/ns/auth/cert#exponent "65537"^^http://www.w3.org/2001/XMLSchema#integer}
+```
+
+As evident, the Blank node is now changed to something more readable and useful. 
+
+The user can transform his key graph into a one of several well-known formats before publishing it on the server. One such format is turtle. In order to do that however, more external libraries are required:
+
+```scala
+import org.w3.banana._
+import org.w3.banana.syntax._
+import ops._  
+import org.w3.banana.jena.Jena
+```
+One can then represent the rdf of the key in turtle format: 
+
+```scala
+@ val toTurtle = turtleWriter.asString(finalKeyPGGraph.graph,"").get
+
+toTurtle: String = """<#key>  <http://www.w3.org/ns/auth/cert#exponent>
+                65537 ;
+        <http://www.w3.org/ns/auth/cert#modulus>
+                "0087acb657366c6a911a0ce2470c24b8d85dd21ae76c6001db37d122c2eafe0fe2ae35541ccdd3c1c81603d98dd7d61b2c31c8605f81fbc3566604b7755793698836caaa99d868477c46b5b735529b50c0acfea6e10fc9b953697a67d57499801beb651e7e08343131a00d1873ab753ce5e79fc961874ee132472f7f210bc38966081fe263c620b67469b52cc555a26dee4fa10d8f40959e0e13516cc0bd1c1669ce53367d28248149142429b127f01e13d9bf21de52a4ac5694d5038a94178e144823b152fa19cbbc094dd40ddbd41e2195a19081125887fccedae21c50660c629a6f2ba4c85a54e1a6472f90b7b62ac7fe58fdae9daa0edc8edec49802fbfac7"^^<http://www.w3.org/2001/XMLSchema#hexBinary> .
+"""
+``` 
+This will return a String representation of the Pointed Graph in turtle format. This representation can also be saved on your local file system like so:
+
+```scala
+write(wd/"publicKey.ttl", RSAKeys.save(toTurtle))
```

Comment:
I think you can't pass a string to `RSAKeys.save` it either takes a public or private key.